### PR TITLE
Couple of enhancements: configurable ssl_verify and returning vm details results as a list of dicts

### PR DESCRIPTION
**Pull request by emptywee**
This PR allows disabling verification of SSL cert when connecting to a vSphere Server. Most of them have a self-signed or outdated certs, so sometimes it's handy to be able to disable it. Can be enhanced to be configurable per vsphere server. Currently it's affecting all vsphere objects defined in the `config.yaml` file.
Also, when you query for hw details of some VM you'll get a list of objects instead of strings. Which is useful, since you can then access queried data as normal.

### CODE REVIEW

**Comment by LindsayHill** [comment]
Hey @emptywee thanks for the PR. I've just fixed a small problem with our new CircleCI setup. It should now run CI checks on PR. Could you push some minor change to this PR, to get it to trigger? Thanks!

**Comment by paul-mulvihill** [comment]
Like the premise. I think it may be worth having the ssl_verify flag within each vsphere server from the start if it's being added. Otherwise you have to disable it on all if only one of them needs it. Which limits it's security if you have one ST2 instance crossing different vsphere environments.

Also can you add this into the CHANGES.md file. I couldn't see an update in there.

Dunno if it's also worth switching this to the new config schema with the change. If not i can look at that part in the new year.

**Comment by LindsayHill** [comment]
👍 for moving to `config.schema.yaml`. It's a little trickier to do for this particular config, so probably better to do it as another PR.

**Comment by emptywee** [comment]
@LindsayHill Absolutely, I will update the change log and add it to the PR.
@paul-mulvihill I will leave it (the per vsphere ssl_verify configuration option) to you as I am not a python developer per se :) I updated the pack to unblock myself from working with our vsphere servers here, and decided to share the idea, hoping that the original author/maintainer (that happens to be you) will re-factor as required.

**Comment by emptywee** [comment]
Oh my gosh, it was easier to add those features than circumvent Circle CI requirements for code formatting :-)

**Comment by emptywee** [comment]
Lol, I already hate it!

**Comment by emptywee** [comment]
I give up!

```
==================== pylint ====================

. virtualenv/bin/activate; if [ ! "actions/vm_hw_details_get.py actions/vmwarelib/actions.py actions/vmwarelib/serialize.py" ]; then echo "No files have changed, skipping run..."; else (REQUIREMENTS_DIR=/home/ubuntu/ci/.circle/ CONFIG_DIR=/home/ubuntu/ci/lint-configs/ st2-check-pylint-pack /home/ubuntu/stackstorm-vsphere || exit 1); fi;
Running pylint on pack: stackstorm-vsphere
************* Module vmwarelib.serialize
E:  6, 4: An attribute defined in json.encoder line 162 hides this method (method-hidden)
make: *** [.pylint] Error 1
```

Not sure if it's within the scope of the changes I made.

File: pack.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-author : Paul Mulvihill
-email : paul.mulvihill@pulsant.com
\ No newline at end of file
+name: vsphere 
+description: st2 content pack containing vsphere integrations.
+version: 0.4.4
+author: Paul Mulvihill
+email: paul.mulvihill@pulsant.com
+contributors:
+  - Igor Cherkaev <emptywee@gmail.com>
```
Comment (Kami) [inline pack.yaml:9]: Nice, good to see your are utilizing new features :)

**Comment by Kami** [comment]
I will go ahead, merge this into master and make lint fixes directly into master.

Thanks again!

**Comment by Kami** [comment]
I merged PR into master and fixed the lint issue, but it would still be great to improve the things @LindsayHill (config schema) and @paul-mulvihill (allowing user to also specify `ssl_verify` per server definition and not just globally) have mentioned above.

**Pull request merged**